reasoner/controller/ConcludableController.java

```diff
@@ -326,7 +377,11 @@ public void rootPull() {
             protected Publisher<Explanation> transformInput(Publisher<PartialExplanation> input,
                                                             Unifier unifier,
                                                             Unifier.Requirements.Instance requirements) {
-                return input.map(
+                return input.flatMap(p -> {
+                    FunctionalIterator<ConceptMap> exists = unifier.unUnify(p.conclusionAnswer().concepts(), requirements);
+                    if (exists.hasNext()) return iterate(p);
+                    else return empty();
+                }).map(
```

Comment:
We should only return an explanation if the answer it contains can actually answer this concludable, which requires that it can ununify.